src/controllers/ElementsController.php

```diff
@@ -1447,6 +1447,57 @@ public function actionGetElementHtml(): Response
         return $this->asJson(compact('html', 'headHtml'));
     }
 
+    /**
+     * Returns HTML for the elements field input that has "relateAncestors" set to true,
+     * based on a given list of selected element ids.
+     *
+     * @return Response
+     * @throws BadRequestHttpException
+     * @throws ForbiddenHttpException
+     * @since 4.0.0
+     */
+    public function actionInputHtml(): Response
+    {
```

Comment:
> that has "relateAncestors" set to true

@myleshyson I might be missing something, but shouldn't this have this `relateAncestors` condition somewhere?